swagger-client-templates/pom.xml

```diff
@@ -31,6 +31,47 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>com.googlecode.maven-download-plugin</groupId>
+                <artifactId>download-maven-plugin</artifactId>
+                <version>1.4.2</version>
+                <executions>
+                    <execution>
+                        <id>fetch-swagger-templates</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>wget</goal>
+                        </goals>
+                        <configuration>
+                            <url>https://github.com/swagger-api/swagger-codegen-generators/archive/v1.0.14.zip</url>
```

Comment:
Maybe tokenize the version number and put it in a property so that when we update in future we only need to change one place. Here and line 68 could both use it.